### ISSUE TITLE
Can't Set Grub Default

### ISSUE THREAD

**Issue by NutmegBurrito** [holoiso-eol/holoiso#480]
## Your Issue:
Title
*type your problem here*
When I try grub customizer to set linux-holoiso as my default (linux-neptune does not boot for my system), I get the following coment:

grub-mconfig couldn't be executed successfully. error message: /etc/default/grub: line 16: syntax error near unexpected token 'elif'

Below is the image of the grub file.

![image](https://user-images.githubusercontent.com/111614158/186983555-50357574-f9b1-4d40-a372-2f3d84911aea.png)

Does anyone know what is wrong with my line 16 and what I need to change?

## Your device's full specs

CPU: Intel i7 11th gen
GPU: NVIDIA GeForce 2070 Super

## Your OS version (`cat /etc/os-release | grep VARIANT_ID | cut -d '"' -f 2`):
5.18.1-arch1_testHoloISO_20220606.1811

**Comment by jrcharney** [comment]
Were you using `grub-customizer` to modify the order of the kernels in GRUB like I did recently?

Don't use that program.  In fact, **DELETE IT!**

```
sudo pacman -Rs grub-customizer
```

The good news is that I had a backup of `/etc/default/grub` on another computer that `grub-customizer` was not installed on.

Replace the contents of `/etc/default/grub` with this to fix it.

```
GRUB_TIMEOUT=5

# Device specific configs
DEVICENAME=$(cat /sys/devices/virtual/dmi/id/product_name)
# OXP Devices
OXP_LIST="ONE XPLAYER:ONEXPLAYER 1 T08:ONEXPLAYER 1S A08:ONEXPLAYER 1S T08:ONEXPLAYER mini A07:ONEXPLAYER mini GA72:ONEXPLAYER mini GT72:ONEXPLAYER GUNDAM GA72:ONEXPLAYER 2 ARP23"
# AYANEO AIR Devices
AIR_LIST="AIR:AIR Pro"
# AYANEO Gen 1 Devices
AYA_LIST="Next:Some fucking thing i don't own right now"

# Define default cmdlines for each device family
if [ "${DEVICENAME}" == "Win600" ]; then
        GRUB_CMDLINE_LINUX_DEFAULT="video=efifb fbcon=rotate:1 quiet splash nvidia-drm.modeset=1 loglevel=3 rd.udev.log_priority=3 vt.global_cursor_default=0"
elif [[ ":$AIR_LIST:" =~ ":$DEVICENAME:"  ]]; then
        GRUB_CMDLINE_LINUX_DEFAULT="video=efifb fbcon=rotate:3 quiet splash nvidia-drm.modeset=1 loglevel=3 rd.udev.log_priority=3 vt.global_cursor_default=0"
elif [ "${DEVICENAME}" == "Jupiter 1" ]; then
        GRUB_CMDLINE_LINUX_DEFAULT="video=efifb fbcon=rotate:1 quiet splash nvidia-drm.modeset=1 loglevel=3 rd.udev.log_priority=3 vt.global_cursor_default=0"
elif [[ ":$OXP_LIST:" =~ ":$DEVICENAME:"  ]]; then
        GRUB_CMDLINE_LINUX_DEFAULT="video=efifb fbcon=rotate:3 quiet splash nvidia-drm.modeset=1 loglevel=3 rd.udev.log_priority=3 vt.global_cursor_default=0"
else
        GRUB_CMDLINE_LINUX_DEFAULT="quiet splash nvidia-drm.modeset=1 loglevel=3 rd.udev.log_priority=3 vt.global_cursor_default=0"
fi

# Define "breeze" Steam Deck GRUB Theme here
GRUB_THEME="/usr/share/grub/themes/breeze/theme.txt"
```

**Comment by NutmegBurrito** [comment]
Thank you very much for the help. Absolutely, I tried to mess when GRUB when I shouldn't of. I was able to delete grub-customizer and fix my /etc/default/grub. 

Is there a way to set linux-holoiso as the default in grub over the linux-neptune? Linux-neptune does not work on my device and wanted to keep my computer as a tv game console without having to use a keyboard to start it each time. If there is no way to do this, its perfectly fine. Again, thank you for the help on this.
